examples/old_symbolics/symbolic_wave_op.py

```diff
@@ -0,0 +1,166 @@
+"""(Old world) symbolic wave equation operator."""
```

Comment:
This file is just a plain copy of the previous symbolic wave equation operator. (for `dagrt-fusion.py`)